examples/ethersv5-ts-esm/src/index.ts

```diff
@@ -1,39 +1,83 @@
-import { wrap } from "@oasisprotocol/sapphire-paratime";
+import { wrapEthersProvider, wrapEthersSigner, wrap } from "@oasisprotocol/sapphire-paratime";
 
-import { BigNumber, ContractFactory, ethers } from "ethers";
+import { ContractFactory, Wallet, providers, Contract, ethers } from "ethers";
 
-import { TestErc20Token } from "./TestErc20Token.js";
 import { assert } from "console";
 
+import OmnibusJSON from "../../../contracts/artifacts/contracts/tests/Omnibus.sol/Omnibus.json" assert { type: "json" };
+
+async function testTheContract(contract:Contract, signerAddr?:string)
+{
+    const addr = await contract.callStatic["testSignedQueries()"]!();
+    if( signerAddr ) {
+        assert( addr === signerAddr );
+    }
+    else {
+        assert( addr === ethers.constants.AddressZero );
+    }
+
+    // XXX: Note that Ethers v5 has a 'workaround' that confuses types
+    //      it makes no distinction between a reverted call returning reversion data
+    //      vs a sucessful call return data with the expected return type.
+    /*
+    try {
+        await contract.callStatic['testCustomRevert()']!();
+        assert(false);
+    }
+    catch(e:any) {
+        console.log(e);
+        assert(e.code == 'CALL_EXCEPTION');
+        assert(e.errorName == 'CustomError');
+        assert((e.errorArgs[0] as BigNumber).toHexString()  == '0x1023456789abcdef1023456789abcdef1023456789abcdef1023456789abcdef');
+    }
+    */
+}
```

Comment:
Should we keep this dead code in?

Reply:
Yes please, it's a tombstone to keep note that this specific behavior is known to be broken in Ethers v5. Where the ethers v5 & v6 tests are carbon copies of each other.